model/stripe/model.sql

```diff
@@ -0,0 +1,71 @@
+create or replace view {{env.schema}}.stripe_invoices as (
+
+  select
+    customer,
+    date,
+    forgiven,
+    subscription as subscription_id,
+    paid,
+    total
```

Comment:
@jthandy looks fine, but wanted to confirm that we want to use the total field, and not amount_due. Total presupposes the discount object.